conan/tools/meson/meson.py

```diff
@@ -1,6 +1,7 @@
 import os
 
 from conans.client.build.cppstd_flags import cppstd_from_settings
+from conans.client.tools.oss import cross_building, get_cross_building_settings
```

Comment:
We need to move this to the new context and profile-build, plus move needed tools to the new ``conan.tool`` namespace. But it is good, definitely not in this PR.